src/types/runtime.go

```diff
@@ -41,6 +41,8 @@ type ZarfCreateOptions struct {
 	SkipSBOM        bool              `json:"skipSBOM" jsonschema:"description=Disable the generation of SBOM materials during package creation"`
 	Insecure        bool              `json:"insecure" jsonschema:"description=Disable the need for shasum validations when pulling down files from the internet"`
 	OutputDirectory string            `json:"outputDirectory" jsonschema:"description=Location where the finalized Zarf package will be placed"`
+	SBOM            bool              `json:"sbom" jsonschema:"description=Whether to pause to allow for viewing the SBOM post-creation"`
+	SBOMOutput      string            `json:"sbomOutput" jsonschema:"description=Location to output an SBOM into after package creation"`
```

Comment:
thoughts on renaming these variables to something more like `SBOMOutputDir` and `ViewSBOM` ?

Reply:
My main thought was that `SBOM bool` looks like it would just be the opposite of `SkipSBOM bool`

Reply:
Yeah I wasn't comfortable with this either, will update.